testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/HttpClientTestOptions.java

```diff
@@ -131,6 +133,7 @@ default Builder withDefaults() {
           .setTestCallbackWithParent(true)
           .setTestErrorWithCallback(true)
           .setTestNonStandardHttpMethod(true)
+          .setTestClientError(true)
```

Comment:
if a test is always enabled then there is no need to add an option to disable it

Reply:
I  do not get it . How can i make the  TestClientError always true without  `setTestClientError(true)`

Reply:
removed